drivers/platform/x86/thinkpad-wmi.c

```diff
@@ -1138,7 +1138,7 @@ static int thinkpad_wmi_debugfs_init(struct thinkpad_wmi *thinkpad)
 				 thinkpad->debug.root,
 				 &thinkpad->debug.instance);
 
-	dent = debugfs_create_u32("instances_count", S_IRUGO,
+	debugfs_create_u32("instances_count", S_IRUGO,
 				 thinkpad->debug.root,
 				 &thinkpad->debug.instances_count);
 	if (!dent)
```

Comment:
I think you want to remove this too no ?

Reply:
sorry I'm a bit confused - isn't line 1141 where I already removed `dent = `?